SECURITY.md

```diff
@@ -0,0 +1,32 @@
+# Security Policy
+
+This project is still under development, the primary goal at this stage is to implement features but we also take security very seriously. This document defines the policy on how to report vulnerabilities and receive updates when patches to those are released.
+
+[Join Nervos Security Team!](mailto:careers@nervos.org)
+
+## Reporting a vulnerability
+
+All security bugs should be reported by sending email to [Nervos Security Team \<security@nervos.org>](mailto:security@nervos.org). Please encrypt your mail using GPG with the following public key.
+
+* Unique ID: `Nervos Security Team <security@nervos.org>`
+* Fingerprint: C3D9 CF8A 1450 D04B 997E  4E31 6CBD D93A 0C9F 6BCD
+* Import from keys.openpgp.org: [0C9F6BCD](https://keys.openpgp.org/search?q=security@nervos.org)
+
+This will deliver a message to Nervos Security Team who handle security issues. Your report will be acknowledged within 24 hours, and you'll receive a more detailed response to your email within 72 hours indicating the next steps in handling your report.
```

Comment:
Nervos Security Team will receive a message and handle the issue. Your report will be acknowledged within 24 hours, and you'll receive a more detailed response within 72 hours informing you of the next steps.